eclipse/hono-code-style.xml

```diff
@@ -176,7 +176,7 @@
 <setting id="org.eclipse.jdt.core.formatter.parentheses_positions_in_try_clause" value="common_lines"/>
 <setting id="org.eclipse.jdt.core.formatter.alignment_for_selector_in_method_invocation" value="16"/>
 <setting id="org.eclipse.jdt.core.formatter.never_indent_block_comments_on_first_column" value="false"/>
-<setting id="org.eclipse.jdt.core.compiler.source" value="1.8"/>
+<setting id="org.eclipse.jdt.core.compiler.source" value="11"/>
```

Comment:
I have set the compliance level in Eclipse to Java 11. When I then export our code style, the file does not contain any of the lines indicating a particular Java version. I suspect that these lines are only added if the compliance level is different from the level of the JDK used for building the project. Can you verify if simply removing these lines (instead of replacing `1.8` with `11` works with your plugin in IDEA as well?

Reply:
Yes. That works. 
Note: In the setting of Eclipse Code Formatter plugin for IntelliJ IDEA the "Java Formatter Version" must be > Eclipse 4.5.0

Reply:
Can you then please update your PR accordingly, i.e. remove those lines completely?

Reply:
Sure. Have you checked if anything else is changed when you export the code style?

Reply:
No, I haven't.